components/GraphDiagram/graph_util.ts

```diff
@@ -278,11 +279,12 @@ export function foreachUpstream(
   callback: (module: Module) => void,
   seen: Set<Module> = new Set(),
 ) {
-  const info = graph.moduleInfos.get(module.key);
-  if (!info || seen.has(module)) return;
   seen.add(module);
+  const info = graph.moduleInfos.get(module.key);
+  if (!info) return;
 
   for (const { module } of info.upstream) {
+    if (seen.has(module)) continue;
```

Comment:
This change is in `foreachUpstream` to match the changes in `foreachDownstream` (which affects the calculation). This function doesn't seem to be used, but I updated to make it consistent.

Reply:
Yup, this method isn't used.  I probably added it thinking I'd have a use for it.  I don't, so just removed the method.